deployments/values/supported-matrix.yaml

```diff
@@ -41,11 +41,11 @@ matrix:
       version: "2"
       pattern: *development_pattern
     intelligence:
-      version: ">=3.0.0-0"
+      version: ">=3.1.0-0"
     trouter:
-      version: ">=4.0.0-0"
+      version: ">=4.1.0-0"
     sfs:
-      version: ">=4.0.0-0"
+      version: ">=4.1.0-0"
```

Comment:
not really necessary, we can even put `>=0.0.0-0` here if you think is more clear or keep as it is